src/api/EscargotPublic.cpp

```diff
@@ -1146,6 +1146,62 @@ Evaluator::EvaluatorResult VMInstanceRef::executePendingJob()
 
 #ifdef ESCARGOT_DEBUGGER
 
+void DebuggerOperationsRef::BreakpointOperations::getStackTrace(DebuggerOperationsRef::DebuggerStackTraceDataVector& outStackTrace)
+{
+    ExecutionState* state = toImpl(m_executionState);
+    SandBox::StackTraceDataVector stackTraceData;
+
+    bool hasSavedStackTrace = SandBox::createStackTraceData(stackTraceData, *state, true);
+    ByteCodeLOCDataMap locMap;
+    size_t size = stackTraceData.size();
+
+    outStackTrace.clear();
```

Comment:
Is this `clear` necessary here?

Reply:
Not necessary. I thought that since the array is passed externally, it might be not empty. Shall I remove it?

Reply:
I think that its better to invoke clear method here as you said.